src/feature/dashboard/presentation/hooks/useAuthorsData.hook.tsx

```diff
@@ -0,0 +1,23 @@
+// import axios from 'axios';
+import { useEffect, useState } from 'react';
+import { IAuthor } from 'core/types/author.types';
+import { authors as mockedAuthors } from '../../data/mockedData';
```

Comment:
Please change this direct import to the flow as in auth feature `mocked data -> data source -> hook`